src/views/HomePage.js

```diff
@@ -25,6 +25,17 @@ class HomePage extends React.Component {
                 </Jumbotron>
               </Col>
             </Row>
+            <Card style={{ width: '18rem' }}>
+            <Card.Img variant="top" src="https://picsum.photos/id/281/200/175" />
```

Comment:
This is a good example of indentation that makes it harder to read. `Card.Img`/`Card.Body` belong inside the `Card`, but that's not clear from indentation.